Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/InputRecordingControls.cs

```diff
@@ -20,11 +21,16 @@ private InputRecordingService RecordingService
         {
             get
             {
-                if (recordingService == null)
+                try
```

Comment:
Interesting, could you provide a little context around why you need this Try/Catch here? Perhaps it was unintentionally added?

Reply:
Again related to issue #6735. After switching profiles with the LoadProfilesOnStartup script, I get a number of errors. In this particular case, I assume that CoreServices.InputSystem is briefly null causing an error when trying to access GetDataProvider.

Reply:
It's related to issue #6735. After switching profiles with the LoadProfilesOnStartup script, I get a number of errors. In this particular case, I got some NullReferenceException. I assume that CoreServices.InputSystem is briefly null causing an error when trying to access GetDataProvider.

Reply:
Can we do a null check for `CoreServices.InputSystem` instead of a `Try/Catch`?

Reply:
It seems like this should be fixed by #7275 once that is in, so perhaps not need to add the workaround?